tests/PerformanceTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Spatie\DataTransferObject\Tests;
+
+use Spatie\DataTransferObject\DataTransferObject;
+
+class MyDTO extends DataTransferObject {
+    /** @var int */
+    public $i;
+}
+
+class PerformanceTest extends TestCase
+{
+    /** @test */
+    public function test()
+    {
+        foreach (range(1, 500000) as $i) {
+            new MyDTO([
+                'i' => $i,
+            ]);
+        }
+
+        $this->markTestSucceeded();
```

Comment:
So, this test :-)

I did notice it in https://github.com/spatie/data-transfer-object/pull/77/files#diff-9a4f6aa431675cae674bd160ff515ab2L17-L20 but was oblivious of its impact.

To give some numbers:
- the whole test suite runs in <500ms on my system (excluding this test)
- this particular test _by itself_ takes minutes; I did stop it after CPU starting to heat up

Question:
- does the test really make sense?
  Maybe benchmarking tests should be handled different, e.g. https://github.com/nuwave/lighthouse uses phpbench and a dedicate stage on travis for it: https://github.com/nuwave/lighthouse/blob/172faba3d9fc7da0e8e7394072c281ce7a7d7b9a/.travis.yml#L65-L69
- does it make sense to part of the default test suite?

Thanks!

Reply:
Feel free to remove it!

Reply:
Done, see https://github.com/spatie/data-transfer-object/pull/85 !